pwndbg/wrappers/readelf.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import pwndbg.wrappers
+
+from pwndbg.wrappers import readelf_path
+
+cmd_opt = {
+           "relocs":"-r",
+           "header":"-h"
+          }
+
+@pwndbg.wrappers.OnlyWithFile(readelf_path)
+def get_jmpslots():
+    readelf_out = subprocess.check_output([readelf_path,cmd_opt["relocs"],pwndbg.file.get_file(pwndbg.proc.exe)]).decode('utf-8')
```

Comment:
Split this into a few lines, this is pretty impossible to read.    Also, I'd prefer just passing `"--relocs"` and `"--file-header"` rather than a lookup for the short version.